File: Aplicaciones Distribuidas/PARCIAL2/04. WEB_CON_MVC_DOTNET_GRUPO5/APLICATIVO/WebConMVCDotnetGrupo5/WebConMVCDotnetGrupo5/Models/DatosEncuesta.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebConMVCDotnetGrupo5.Models
{
    public class DatosEncuesta
    {
        private string txtNombre;
        private string txtApellido;
        private string[] chkProgLeng;
        private string imgProfile;

        public string getTxtNombre()
        {
            return txtNombre;
        }

        public void setTxtNombre(string txtNombre)
        {
            this.txtNombre = txtNombre;
        }

        public string getTxtApellido()
        {
            return txtApellido;
        }

        public void setTxtApellido(string txtApellido)
        {
            this.txtApellido = txtApellido;
        }

        public string[] getChkProgLeng()
        {
            return chkProgLeng;
        }

        public void setChkProgLeng(string[] chkProgLeng)
        {
            this.chkProgLeng = chkProgLeng;
        }

        public string getImgProfile()
        {
            return imgProfile;
        }

        public void setImgProfile(string imgProfile)
        {
            this.imgProfile = imgProfile;
        }
    }
}

File: Aplicaciones Distribuidas/PARCIAL1/03. WEB_CON_MVC_JAVA_CHUQUIMARCA/APLICATIVO/WebConMVCJavaGrupo5/src/java/ec/edu/monster/modelo/DatosEncuesta.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/J2EE/EJB30/StatelessEjbClass.java to edit this template
 */
package ec.edu.monster.modelo;

import javax.ejb.Stateless;

/**
 *
 * @author santi
 */
@Stateless
public class DatosEncuesta {

    private String txtNombre;
    private String txtApellido;
    private String[] chkProgLeng;
    private String imgProfile;

    public DatosEncuesta() {
    }

    public String getTxtNombre() {
        return txtNombre;
    }

    public void setTxtNombre(String txtNombre) {
        this.txtNombre = txtNombre;
    }

    public String getTxtApellido() {
        return txtApellido;
    }

    public void setTxtApellido(String txtApellido) {
        this.txtApellido = txtApellido;
    }

    public String[] getChkProgLeng() {
        return chkProgLeng;
    }

    public void setChkProgLeng(String[] chkProgLeng) {
        this.chkProgLeng = chkProgLeng;
    }

    public String getImgProfile() {
        return imgProfile;
    }

    public void setImgProfile(String imgProfile) {
        this.imgProfile = imgProfile;
    }

}
